pytest_itde/__init__.py

```diff
@@ -166,7 +166,8 @@ def start_db(name, itde, exasol, bucketfs):
             environment_name=name,
             database_port_forward=exasol.port,
             bucketfs_port_forward=bucketfs_url.port,
-            ssh_port_forward=20002,
+            # should this rather be passed as a parameter to _bootstrap_db()?
+            ssh_port_forward=Ports.forward.ssh,
```

Comment:
yes, it should

Reply:
Should it be a separate option group then or can we add it to group `EXASOL`?

Reply:
We have different option groups for DB and BucketFS. So, I think, we should a separate one for ssh, too.

Reply:
See next push